package.nls.json

```diff
@@ -14,6 +14,7 @@
   "contributes.command.pw.extension.command.toggleModels": "Toggle Playwright configs",
   "contributes.command.pw.extension.toggle.reuseBrowser": "Toggle Playwright browser reuse",
   "contributes.command.pw.extension.toggle.showTrace": "Toggle Playwright Trace Viewer",
+  "contributes.command.pw.extension.toggle.embeddedTraceViewer": "Toggle Embedded Playwright Trace Viewer",
```

Comment:
```suggestion
  "configuration.playwright.embeddedTraceViewer": "Toggle Embedded Playwright Trace Viewer",
```

Reply:
Addressed in b8a3d8b105bfd17b1c4c1dd8b4263723356ef219